### DISCUSSION TITLE
Accuracy of the model

### DISCUSSION THREAD

**Discussion by Aakasha01Agarwal** [Xilinx/brevitas-radioml-challenge-21#17] [Training]
Hello, what is the precision of the accuracy that our model should have. Will an accuracy of 56.07% considered eligible?

**Comment by maltanar** [comment]
Hi @Aakasha01Agarwal , quoting the evaluation criteria section of the [challenge website](https://challenge.aiforgood.itu.int/match/matchitem/34) here:

> Must reach at least 56.000% accuracy averaged across all modulations and all SNRs present in RadioML 2018.01A on the test set (for the provided train-test split in the sandbox). The accuracy will be measured (using the Docker container) on the submitted trained .pth checkpoint and code. Extra accuracy over this threshold will not bring benefits for the score.

So 56.07% is sufficient, but you need to make sure that we can reproduce this accuracy (as stated above). If your accuracy is higher this might bring some extra safety, e.g. sometimes CPU vs GPU inference can yield a slight difference in accuracy.

**Reply by Aakasha01Agarwal** [reply]
Thanks, @maltanar for such a quick response. So if I provide you a docker container with my code and the model (and everything required), you will run the code as it is without changing it anyhow, is that right or are there any other checks that you guys will perform?

PS. My code contains the methods that you guys provided for inference cost calculations.
